tests/acceptance/WPFirstCest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+
+class WPFirstCest
```

Comment:
Two questions:

1. What does this name mean?
2. Why doesn't this test subclass some kind of test class? What is the point of all those `_support` classes if they wouldn't be used in a case like this?